numpy/core/tests/test_longdouble.py

```diff
@@ -80,6 +81,11 @@ def test_fromstring():
                  err_msg="reading '%s'" % s)
 
 
+@in_foreign_locale
+def test_fromstring_best_effort():
+    assert_equal(np.fromstring("1,234", dtype=float, sep=" "),
```

Comment:
Heh, I didn't expect that, I would have expected an error. I assume what is being checked is that `,` is not handled as the radix character?